corehq/blobs/pillow.py

```diff
@@ -19,46 +14,28 @@
 
 class BlobDeletionProcessor(PillowProcessor):
 
-    def __init__(self, blob_db, bucket_base):
+    def __init__(self, blob_db):
         super(BlobDeletionProcessor, self).__init__()
         self.blob_db = blob_db
-        self.bucket_base = bucket_base
 
     def process_change(self, pillow_instance, change):
         if change.deleted:
-            bucket = join(self.bucket_base, change.id)
+            bucket = "{}/{}".format(change.meta.data_source_name, change.id)
```

Comment:
> But for now I think this should work as long as we update the kafka feed to publish hard deletions

Do you mean "update the kafka feed to *only* publish hard deletions" as in *do not push soft deletions*?

> and don't process deletions from forms.

It's a unclear why we wouldn't want to delete forms if the corresponding data in couch/postgres was being hard-deleted. I'm not sure how we'd ever use blobs that were not referenced by a record in another database.